HW2.js

```diff
@@ -0,0 +1,9 @@
+var numArr = process.argv;
+total = 0;
```

Comment:
Don't forget to declare your variable using the var keyword. i.e. `var total = 0;`